src/services/context_menu/context_menu_service.cpp

```diff
@@ -229,6 +233,65 @@ namespace big
 			got_an_entity |= get_closest_to_center(pools::get_all_peds);
 			got_an_entity |= get_closest_to_center(pools::get_all_props);
```

Comment:
These calls to `get_closest_to_center` can have their return values modified to m_veh_pointer, m_ped_pointer, m_prop_pointer.
You can then also move those member values to normal stack values within the function (aka not part of the class).